analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -153,12 +153,12 @@ def add_arguments_to_parser(parser):
     Add the subcommand's arguments to the given argparse.ArgumentParser.
     """
 
-    parser.add_argument('logfile',
-                        type=str,
-                        help="Path to the JSON compilation command database "
-                             "files which were created during the build. "
-                             "The analyzers will check only the files "
-                             "registered in these build databases.")
+    parser.add_argument('input',
```

Comment:
`CodeChecker check` command has a slightly different interface, for example it doesn't have this positional argument. It can be given the compilation database through `-l` flag, or a build command can be given by `-b` flag.